hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupManifest.java

```diff
@@ -438,6 +438,7 @@ public BackupManifest(FileSystem fs, Path backupPath) throws BackupException {
           long len = subFile.getLen();
           byte[] pbBytes = new byte[(int) len];
           in.readFully(pbBytes);
+          in.close();
```

Comment:
Better use the try-with-resources grammar?

Reply:
Oh,yes, using the try-with-resources grammar is better. Thanks for reviewing the code.